### PR TITLE
Add contents to readme file

### PR DESCRIPTION
**Pull request by LouiaR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,3 +31,12 @@ Run project locally:
 cd frontend && yarn start
 cd backend && yarn start
 ```
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
+
+* Hat tip to anyone who's code was used
```
Comment (kkarimi) [inline README.md]: This section seems to be a bit off, maybe credit Code Your Future mentors and students for now.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+* This repo is frontend and backend
+* It requires NODE
+* This app is built with:
+REACT 
+Material UI
+SQLITE for database...
 
----
 
 [![CircleCI](https://circleci.com/gh/CodeYourFuture/directory-of-services/tree/master.svg?style=svg)](https://circleci.com/gh/CodeYourFuture/directory-of-services/tree/master)
```
Comment (kkarimi) [inline README.md]: This badge is usually towards the top

File: README.md [hunk 3]
```diff
@@ -1,12 +1,23 @@
...
+# Directory of Services 
+
+This App is destined primary for case workers who provide services oriented for refugees, asylum seekers and organisations working with them to identify services that offer advice and support to assist refugees and asylum seekers rebuild, improve their lives and facilitate their insertion inside the society.
+
+## Getting Started
+
+
+* This repo is frontend and backend
+* It requires NODE
+* This app is built with:
```
Comment (kkarimi) [inline README.md]: Can you add the ER diagram? I think Eugene has a pretty version

File: README.md (hunk 1 above)
Comment (LouiaR) [reply README.md]: Ok , got it bro.

**Pull request merged**